Sito/back/gestione_profilo_admin.html

```diff
@@ -31,7 +31,7 @@ <h1 xml:lang="ja" lang="ja">Sushi Nakamura</h1>
       <p>Ti trovi in: <a href="home_admin.html" xml:lang="en" lang="en">Home</a> &gt; &gt; Gestione Profilo </p>
     </div>
 
-    <div class="content" id="gestione_admin">
+    <div class="content" id="gestione_profilo">
```

Comment:
Modificare id in modo che sia uguale a quello per l'aiuto degli screen reader e creare una classe in comune con i due form per avere lo stesso css

Reply:
Creo una issue per unificare i CSS dei vari form